nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -560,6 +560,10 @@ define([], () => {
                     mode: 'error',
                     stage: 'launching',
                 },
+                {
+                    mode: 'error',
+                    stage: 'runtime',
+                },
```

Comment:
could not transition from the launch state to an error ==> added in the error state as a valid next option